### ISSUE TITLE
Fix Frontend Failing Test: numpy - loss.paddle.nn.functional.binary_cross_entropy

### ISSUE THREAD

**Issue by ivy-leaves** [unifyai/ivy#27717]
### Test Details
- **Function**: `paddle.nn.functional.binary_cross_entropy`
- **Library**: numpy
- **Submodule**: loss

### Test Status
- **Status**: Fail
- **Workflow**: [Link to Workflow](https://github.com/unifyai/ivy/actions/runs/7117436922/job/19378157041)
- **Test Path**: `ivy_tests/test_ivy/test_frontends/test_paddle/test_nn/test_functional/test_loss.py::test_paddle_binary_cross_entropy`
